src/layer/layer.js

```diff
@@ -0,0 +1,24 @@
+import { Observable } from '../util/observable';
+export class Layer extends Observable {
+    constructor(options) {
+        super();
+        this._source = options.source;
+        this._view = options.view;
+        // this._render = new LineLayerRender(options.context.gl);
+        this._renderList = [];
```

Comment:
Maybe it will be more consistent  after using `_renderObjects` instead of `_renderList`